app/code/Magento/InventoryInStorePickupAdminUi/Plugin/Ui/DataProvider/ConvertBooleanToStringPlugin.php

```diff
@@ -48,12 +48,14 @@ public function afterGetData(
      *
      * @return array
      */
-    private function convertExtensionAttributeBooleanToIntStr(array $item):array {
-        if (isset($item[InStorePickupInterface::EXTENSION_ATTRIBUTES_KEY]) &&
-            isset($item[InStorePickupInterface::EXTENSION_ATTRIBUTES_KEY][InStorePickupInterface::IN_STORE_PICKUP_CODE])
-        ) {
-            $item[InStorePickupInterface::EXTENSION_ATTRIBUTES_KEY][InStorePickupInterface::IN_STORE_PICKUP_CODE] =
-                (string)(int)$item[InStorePickupInterface::EXTENSION_ATTRIBUTES_KEY][InStorePickupInterface::IN_STORE_PICKUP_CODE];
+    private function convertBooleanToString(array $item):array
```

Comment:
Method name is slightly misleading. It accepts `array` but is supposed to covert `bool` to `string`.

Reply:
Fixed.
Please re-check.